app/web/template/layout/radios.gohtml

```diff
@@ -11,6 +19,21 @@
             {{ tr $.top.App $e.hint }}
           </div>
         {{ end }}
+        {{ if $e.conditionalLabel }}
+            <div class="govuk-radios__conditional" id="{{ fieldID $.name $i }}-conditional-div">
+                <div class="govuk-form-group {{ if index $.top.Errors $.name }}govuk-form-group--error{{ end }}">
+                    <label class="govuk-label" for="{{ $e.conditionalInputName }}">
+                        {{ tr $.top.App $e.conditionalLabel }}
+                    </label>
+                    {{ if index $.top.Errors $e.conditionalInputName }}
+                        <p id="{{ fieldID $.name $i }}-error" class="govuk-error-message">
+                            {{ tr $.top.App (index $.top.Errors $e.conditionalInputName) }}
+                        </p>
+                    {{ end }}
+                    <textarea class="govuk-textarea {{ if index $.top.Errors $.name }}govuk-input--error{{ end }}" id="{{ $e.conditionalInputName }}" name="{{ $e.conditionalInputName }}" rows="10">{{ $e.conditionalValue }}</textarea>
+                </div>
+            </div>
+        {{ end }}
```

Comment:
I think for very customised things like this it would be better not to use this helper template and write out the `govuk-radios` fully, otherwise this template will keep growing with quite specific things

Reply:
Yeah, agree. My other alternative was to break this into a `conditional-item` or `conditional-radio` partial so we could drop some of the logic from the template. Happy to wait until we get our next conditional item though - I don't have a feel yet for how much this pattern will appear.